orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -3977,3 +3979,32 @@ def __init__(self, module, debug_options=None):
     assert prediction is not None
     prediction = prediction.sum()
     prediction.backward()
+
+def test_ortmodule_fused_adam_optimizer():
+
+    device = 'cuda'
+    N, D_in, H, D_out = 64, 784, 500, 10
+    pt_model = NeuralNetSinglePositionalArgument(D_in, H, D_out).to(device)
+
+    ort_model = ORTModule(pt_model)
+    optimizer = FusedAdam(ort_model.parameters())
+
+    def run_step(model, x):
+        prediction = model(x)
+        loss = prediction.sum()
+        loss.backward()
+        return loss
+
+    def run_optim_step(optimizer):
+        optimizer.step()
+        optimizer.zero_grad()
+
+    ga_steps = 2
+    ort_model.zero_grad()
+
+    for step in range(10):
```

Comment:
This can be done in the next PR.
take one pytorch model, run with transformer.AdamW 
Take the cloned ortmodule model, run with ort.fusedadam. 

Compare model.weight, param.grad, loss for every steps. assert the values is close.

Reply:
Updated the unit test to do this.